hwy/ops/generic_ops-inl.h

```diff
@@ -1249,6 +1250,43 @@ HWY_API V PopulationCount(V v) {
 }
 #endif
 
+HWY_API Vec128<int64_t,1> operator*(Vec128<int64_t,1> x, Vec128<int64_t,1> y) {
+    return Set(Simd<int64_t,1,0>(), GetLane(x)*GetLane(y));
+}
+HWY_API Vec128<uint64_t,1> operator*(Vec128<uint64_t,1> x, Vec128<uint64_t,1> y) {
+    return Set(Simd<uint64_t,1,0>(), GetLane(x)*GetLane(y));
+}
+
+#ifndef HWY_NATIVE_I64MULLO
+
+
+HWY_API Vec128<uint64_t> operator*(Vec128<uint64_t> x, Vec128<uint64_t> y) {
+  Vec128<uint32_t> x32 = BitCast(Simd<uint32_t,4,0>(), x);
```

Comment:
To make this more readable I'd recommend a typedef V32 (`using V32 = Vec128<uint32_t>;`) and an lvalue for the Simd: `Full128<uint32_t> d32;`).